pipeline.py

```diff
@@ -91,7 +96,41 @@ def validate_dataset(path_to_validate):
     """
     Validates folder with assets
     """
-    pass
+    path = pathlib.Path(path_to_validate)
+
+    if not path_to_validate.exists():
+        raise FileNotFoundError
+
+    if not path.is_dir():
+        raise NotADirectoryError
+
+    if not path.glob('*'):
+        raise EmptyDirectoryError
+
+    files = {
+        "json": 0,
+        "pdf": 0,
+        "txt": 0
+    }
+    for file in path.iterdir():
+        file_name = file.name
+        if not re.match(r'\d+_(raw\.(txt|pdf)|meta\.json)', file_name):
+            raise InconsistentDatasetError
+
+        file_number = int(re.findall(r'\d+', file_name)[0])
+        current_file_number = files.get(file.suffix)
```

Comment:
what do you want to achieve in this line?

Reply:
is it possible for current_file_number to become None?

Reply:
1) I wanted to check the current id of the file.
2) Yes, it can become `None` in case there will be a file with unexpected file extension